### DISCUSSION TITLE
Schematic

### DISCUSSION THREAD

**Discussion by gabizro** [Tunas1337/UV-K5-Modded-Firmwares#24] [Show and tell]
First try to draw schematics.
RX part.
[RX.pdf](https://github.com/Tunas1337/UV-K5-Modded-Firmwares/files/11735381/RX.pdf)
![layout_back_01_sch](https://github.com/Tunas1337/UV-K5-Modded-Firmwares/assets/7538325/84f9c890-9907-4a20-8a99-3a5c5a927b16)

**Comment by Tunas1337** [comment]
Thanks for moving this! I'll also attach your other schematic to this thread.

**Comment by Tunas1337** [comment]
![image](https://user-images.githubusercontent.com/7538325/245113114-58192ffe-2fe8-4abf-b09f-f37c45e2a4e9.jpg)

**Comment by gabizro** [comment]
Switching threshold on RX path is at 280 MHz
Under 280 MHz Q3 path is active
From 280 MHz up Q4 is active

**Comment by EB4FBZ** [comment]
Q1, Q2: R24 SC-70 on the RX path are 2SC4226
TX pre-driver R24 SOT-23 on the TX path is 2SC3356
RX switching diodes "UD" SOD-523 are probably KDS114E
TX switching diodes "TY" looks like SOD-123 so it's probably 1SS241

Could we change the switching threshold to a different frequency? Making a CB/10m + 6/4m band radio could be interesting.

**Reply by vsh75** [reply]
"TY" means 1SS314

**Comment by vsh75** [comment]
What is the purpose of FET 2301A near the UHF TX filter?

**Comment by gabizro** [comment]
Busy with work so no more time on schematic 
For such wide input 18Mhz to 1.3Ghz I think best is to divide into more bands.
Maybe redesign RX circuit with separate paths or with some LNA and switching input filter circuits.
We may use some GPIO pins from MCU (if any available) but require some firmware modification
Or use some like SN74LVC1G139 and with 2 GPIO pins already used for that on BK4819 we can have 4 outputs , that require only one function mod in firmware.
